client/src/components/ElementEditor/ElementContext.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react';
+
+export const ElementContext = createContext(null);
```

Comment:
Having this in a separate file isn't a useful abstraction if we're not setting any values in the context here.
If there are no default values we can give the context, please just move it back into `Element`.

Reply:
OK will move it back and leave a note in the description of this PR why we're not implementing that AC